flask_project/url_scanner.py

```diff
@@ -31,3 +33,66 @@ def get_domain(url: str) -> str:
     """
     parsed_url = urlparse(url)
     return parsed_url.netloc
+
+def url_validation(url):
+    """
+    Function check if given url name has correct URL structure
+
+    :Parameters:
+        - url (string): String with URL to check
+    :Return:
+        - True/False
+    """
+    validation = urlparse(url)
+    return bool(validation.scheme and validation.netloc and validation.scheme in ['http', 'https'])
+
+def domain_validation(domain):
+    """
+    Function check if given domain name has correct structure
+
+    :Parameters:
+        - domain (string): String with domain name to check
+    :Return:
+        - True/False
+    """
+    regex = re.compile(
+        r'^(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z]{2,63}$'
+    )
+    return True if regex.match(domain) else False
+
+def url_scan_info_check(url_list):
+    """
+    Function return scan info for given URLs names
+
+    :Parameters:
+        - url_list (list): List with URLs
+    :Return:
+        - scan_info (list): List with scan info for each given URLs names
+    """
+    scan_info = []
+    for url in url_list:
+        if url_validation(url) is True:
+            scan_info.append( get_url_scan_info(url))
+        else:
+            scan_info.append([url,"Wrong input - given string is not URL address"])
+    return (scan_info)
```

Comment:
Is it really necessary to wrap scan_info?

Reply:
Issue fixed.